pkg/operators/oci-handler/oci.go

```diff
@@ -96,6 +99,20 @@ func (o *ociHandler) InstanceParams() api.Params {
 			Description: "Secret to use when pulling the gadget image",
 			TypeHint:    api.TypeString,
 		},
+		{
+			Key:          verifyImage,
+			Title:        "Verify image",
+			Description:  "Verify image using the provided public key",
+			DefaultValue: "true",
```

Comment:
I'm not sure whether we should enable this by default - if people want to try out stuff from artifact hub, this might confuse more than it helps.

Reply:
Yeah, you're right, it's easy enough to disable it.

Reply:
I agree it should be done by default.